notebooks/intercomparison-covermaps.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p>nit: I suggest writing this out with some new lines so it's more legible</p><pre class="ql-syntax" spellcheck="false">new_report = pd.DataFrame(data = {
    "dataset": dataset,
    "accuracy": report["accuracy"],
    ....
})
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>